styles/item.less

```diff
@@ -3,19 +3,20 @@
   align-items: center;
   justify-content: space-between;
   margin: 0 0 10px 0;
-  padding: 0 20px;
+  padding: 10px 20px;
   width: 100%;
-  height: 70px;
+  // height: 70px;
   background: @item-color;
   border: none;
   border-radius: 3px;
   box-sizing: border-box;
   font-size: @item-font-size;
   .item-name {
     width: 400px;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-    overflow: hidden;
+    word-break: break-all;
+    &:first-letter {
+      text-transform: uppercase;
```

Comment:
I'd remove this section, the first letter doesn't need to be uppercase (you can see in your screenshot that the node modules path started with an uppercase, which wasn't needed)